src/components/CommentList/CommentList.jsx

```diff
@@ -1 +1,25 @@
-export const CommentList = () => <>Put the list here</>;
+import { CommentInfo } from '../CommentInfo';
+import './CommentList.scss';
+
+export const CommentList = ({ comments }) => {
+  if (!comments.length) {
+    return (
+      <div
+        data-cy="NoCommentsMessage"
+      >
```

Comment:
depends on project conventions, but we usually keep it on the same line if there's only one attribute

```suggestion
      <div data-cy="NoCommentsMessage">
```